### PR TITLE
[DE] Add `Guides/BSS_Issues`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Guides/BSS_Issues/de.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+# Wie man Probleme mit dem BSS löst
+
+Wenn das **[Beatmap Submission System](/wiki/Submission)** (***BSS***) dich nicht deine eigenen Beatmaps einreichen lässt, weil es der Meinung ist, dass die Beatmap nicht dir gehört, dann mache folgendes:
+
+1. Gehe zum Ordner der Beatmap.
+   - Du kannst es leicht erreichen, indem du `Datei` und dann `Songs-Ordner öffnen` im [Beatmap-Editor](/wiki/Client/Beatmap_editor) anklickst.
+2. Beende osu! (Dadurch wird sichergestellt, dass alle Änderungen korrekt übernommen werden.)
+3. Öffne die `.osu`-Dateien für die Schwierigkeitsstufen mit irgendeinem Texteditor (z. B. Notepad).
+4. Gebe deinen Benutzernamen bei `Creator` an. (Wenn dein Name Leerzeichen enthält, dann nutze Leerzeichen.)
```
Comment (Pasi4K5) [inline wiki/Guides/BSS_Issues/de.md:17]:
```suggestion
4. Gib deinen Benutzernamen bei `Creator` an. (Wenn dein Name Leerzeichen enthält, dann nutze Leerzeichen.)
```

File: wiki/Guides/BSS_Issues/de.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+5. Setze `BeatmapID` auf `0`.
+6. Setze `BeatmapSetID` auf `-1`.
+7. Überprüfe, dass der Name des Beatmap-Ordners nicht mit einer Zahlenfolge beginnt.
```
Comment (Pasi4K5) [inline wiki/Guides/BSS_Issues/de.md:20]:
```suggestion
7. Stelle sicher, dass der Name des Beatmap-Ordners nicht mit einer Zahlenfolge beginnt.
```

File: wiki/Guides/BSS_Issues/de.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+Solltest du immer noch Probleme haben, folge diesen Schritten:
+
+1. Exportiere die Beatmap mit Hilfe des Beatmap-Editors.
+   - `Datei`, dann `Paket exportieren`.
+2. Verlasse den Beatmap-Editor und lösche danach die Beatmap in osu!.
+3. Beende osu!
+4. Gehe zur exportierten Beatmap und ändere die Erweiterung von `.osz` auf `.zip` (dazu müssen die Dateierweiterungen in den Windows-Ordneroptionen sichtbar sein).
+5. Extrahiere die `.zip`-Datei.
+6. Öffne die `.osu`-Dateien für die Schwierigkeitsstufen mit irgendeinem Texteditor (z. B. Notepad).
+7. Gebe irgendetwas anderes als Nutzername bei `Creator` ein, aber nutze nicht deinen Benutzernamen.
```
Comment (Pasi4K5) [inline wiki/Guides/BSS_Issues/de.md:34]:
```suggestion
7. Gib irgendetwas als Nutzernamen bei `Creator` ein, aber nutze nicht deinen Benutzernamen.
```

File: wiki/Guides/BSS_Issues/de.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+8. Setze `BeatmapID` auf `0`.
+9. Setze `BeatmapSetID` auf `-1`.
+10. Sobald du fertig bist, packe die Dateien wieder in die `.zip`-Datei.
+11. Nenne die `.zip`-Datei um zu `.osz`.
```
Comment (Pasi4K5) [inline wiki/Guides/BSS_Issues/de.md:38]:
```suggestion
11. Benenne die `.zip`-Datei zu `.osz` um.
```

File: wiki/Guides/BSS_Issues/de.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  - Ich kann meine eigenen Beatmaps nicht einreichen!
+  - Probleme mit dem Beatmap Submission System
+---
+
+# Wie man Probleme mit dem BSS löst
+
+Wenn das **[Beatmap Submission System](/wiki/Submission)** (***BSS***) dich nicht deine eigenen Beatmaps einreichen lässt, weil es der Meinung ist, dass die Beatmap nicht dir gehört, dann mache folgendes:
+
+1. Gehe zum Ordner der Beatmap.
+   - Du kannst es leicht erreichen, indem du `Datei` und dann `Songs-Ordner öffnen` im [Beatmap-Editor](/wiki/Client/Beatmap_editor) anklickst.
```
Comment (FamousPig) [inline wiki/Guides/BSS_Issues/de.md:14]:
```suggestion
   - Du kannst ihn leicht erreichen, indem du `Datei` und dann `Songs-Ordner öffnen` im [Beatmap-Editor](/wiki/Client/Beatmap_editor) anklickst.
```

Alternatively `Das kannst du leicht erreichen`, if that is what you were going for.

**Review by FamousPig — CHANGES_REQUESTED** [review]

File: wiki/Guides/BSS_Issues/de.md (hunk 1 above)
Comment (The-Last-Cookie) [reply wiki/Guides/BSS_Issues/de.md]: eh, mit dem Imperativ habe ich es irgendwie lol

**Review by TicClick — APPROVED** [review]

**Pull request merged**